core/domain/change_domain.py

```diff
@@ -164,6 +181,11 @@ def validate_dict(self, change_dict):
                 valid_cmd_attribute_specs = copy.deepcopy(cmd)
                 break
 
+        for cmd in self.DEPRECATED_COMMANDS:
+            if cmd == cmd_name:
+                raise utils.DeprecatedCommandError(
```

Comment:
Update the docstring for this new raise condition*

Reply:
Done.